Linux/Car_Client/chronos.h

```diff
@@ -20,6 +20,8 @@ class Chronos : public QObject
 
 private slots:
     void startTimerSlot();
+	void abort();
+	void heabTimeOutAbort();
```

Comment:
Where's the implementation of headTimeOutAbort?

Reply:
Good catch, I forgot to remove that definition. Since an abort is an abort I instead connected everything like this in chronos.cpp 38:
```
connect(mChronos, SIGNAL(heabTimeOut()),
			this, SLOT(Chronos::abort()) );
```